### PR TITLE
#161361392 Remove all jQuery Instances from the repository

### PR DESCRIPTION
**Pull request by DrKimpatrick**
#### What does this PR do?
* removes jQuery instances from the repository
* removes window.store from the repository
* adds missing semicolons to code
#### Description of Tasks to be completed?
* performs code clean up to ensure that code is written in pure React
#### What are the relevant pivotal tracker stories?

[#161361392](https://www.pivotaltracker.com/story/show/161361392) 

#### [Heroku Link](https://ah-frontend-staging-pr-27.herokuapp.com/#/)

### CODE REVIEW

File: src/components/Auth/Login/tests/LoginForm.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {mount, shallow} from 'enzyme';
 import $ from 'jquery';
 import {MemoryRouter} from "react-router-dom";
 import {Provider} from "react-redux";
-import {store} from "../../../../redux-js/stores";
+import store from "../../../../redux-js/stores/index";
```
Comment (isheebo) [inline src/components/Auth/Login/tests/LoginForm.test.js]: why are you importing store here?

File: src/components/navigation/NavBar.js [hunk 2]
```diff
@@ -92,10 +91,11 @@ NavBar.propTypes = {
 
 NavBar.defaultProps = {
 	isLoggedIn: false,
-    visible: true,
+    visible: true
 };
-
+console.log({...store});
```
Comment (isheebo) [inline src/components/navigation/NavBar.js]: remove all console.log statements

File: src/redux-js/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import ACTIONS from './actions/action';
-import { store } from './stores';
-
-window.store = store;
-window.requestLoadingAction = ACTIONS.requestLoadingAction;
-window.socialLoginServiceAction = ACTIONS.socialLoginServiceAction;
+// import ACTIONS from './actions/action';
```
Comment (isheebo) [inline src/redux-js/index.js]: remove all commented code. it ain't needed if it is commented out

Comment (DrKimpatrick) [reply src/redux-js/index.js]: I have acted accordingly.

**Comment by isheebo** [comment]
approved

**Pull request merged**